src/applications/financial-status-report/components/DependentAges.jsx

```diff
@@ -0,0 +1,108 @@
+import React, { useState, useEffect, useMemo, useCallback } from 'react';
+import { useSelector, useDispatch } from 'react-redux';
+import { VaTextInput } from '@department-of-veterans-affairs/component-library/dist/react-bindings';
+import { setData } from 'platform/forms-system/src/js/actions';
+import { DEPENDENT_AGE_LABELS } from '../constants/dependentLabels';
+
+export const isNumber = value => {
+  const pattern = /^[0-9]+$/;
+  return pattern.test(value);
+};
```

Comment:
could probably move this to the helpers file

Reply:
@amponce there is a `validations.js` file in the `utils/` folder

Reply:
Added!